test/test_runner.go

```diff
@@ -212,7 +214,18 @@ func (r *TestRunner) RunTest(script string, funcName string) (result *Result, er
 		})
 	}()
 
-	_, inter, err := r.parseCheckAndInterpret(script)
+	program, inter, err := r.parseCheckAndInterpret(script)
```

Comment:
The first time, we parse / check / interpret the test script, in order to perform the contract deployments inside the `init` special function.